platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/AbstractMigrationReportTest.java

```diff
@@ -21,8 +21,8 @@
 import org.junit.Before;
 
 /** Base class for report-type test cases which handles setup of various messages. */
-// squid:S2187 - hopefully the doc above indicates this is a base class for test classes
-@SuppressWarnings("squid:S2187")
+@SuppressWarnings(
+    "squid:S2187" /* hopefully the doc above indicates this is a base class for test classes */)
```

Comment:
✏️  Another "hopefully" style comment. What do we mean by this? Can we remove this?